alfresco/alfresco-content-app/stack.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.1'
+services:
+    app:
+        build: '.'
```

Comment:
This won't work because it's using a `build` instruction and there's nothing to build here.